### DISCUSSION TITLE
Any similar library for reCAPTCHA?

### DISCUSSION THREAD

**Discussion by ghost** [QIN2DIM/hcaptcha-challenger#35] [Q&A]
Is there any similar library that solves reCAPTCHA v2 with tiled images?

**Comment by QIN2DIM** [comment]
The solution ideas for these two challenges are completely different and the difficulty is worlds apart.

**Comment by QIN2DIM** [comment]
Generally processed by speech to text

**Reply by ghost** [reply]
Got it. Thanks!

**Comment by QIN2DIM** [comment]
You can refer to this project that I wrote https://github.com/QIN2DIM/armour-recaptcha

Note that the content in it may be outdated, but the good news is that it still works.
